### PR TITLE
Quick fix for portlayer swagger spec so that it validates against swagger 2.0

### LINKED ISSUE #1840: The swagger spec at "lib/apiservers/portlayer/swagger.yml" is invalid against swagger specification 2.0
```
swagger-linux validate lib/apiservers/portlayer/swagger.yml
The swagger spec at "lib/apiservers/portlayer/swagger.yml" is invalid against swagger specification 2.0. see errors :
- paths./vch/info.get.contains in body is a forbidden property
- paths./vch/info.get.contains in body is a forbidden property
- must validate one and only one schema (oneOf)
- must validate one and only one schema (oneOf)
- must validate one and only one schema (oneOf)
- paths./containers/{id}.delete.parameters.type in body should be one of [string number boolean integer array]
- paths./containers/{id}.delete.parameters.in in body should be one of [header]
- must validate one and only one schema (oneOf)
- must validate one and only one schema (oneOf)
- must validate one and only one schema (oneOf)
- paths./containers/{id}.delete.parameters.in in body should be one of [header]
- paths./containers/{id}.delete.parameters.type in body should be one of [string number boolean integer array]
- must validate one and only one schema (oneOf)
- must validate one and only one schema (oneOf)
- must validate one and only one schema (oneOf)
- paths./containers/{id}.delete.parameters.in in body should be one of [header]
- paths./containers/{id}.delete.parameters.type in body should be one of [string number boolean integer array]
- must validate one and only one schema (oneOf)
- must validate one and only one schema (oneOf)
- must validate one and only one schema (oneOf)
- paths./containers/{id}.delete.parameters.in in body should be one of [header]
- paths./containers/{id}.delete.parameters.type in body should be one of [string number boolean integer array]

```

### PR DESCRIPTION
**Pull request by jzt**
Fixes #1840

### CODE REVIEW

File: lib/apiservers/portlayer/swagger.json [hunk 1]
```diff
@@ -1447,7 +1447,8 @@
 					{
 						"name": "timeout",
 						"in": "query",
-						"type": "int64",
+            "type": "integer",
```
Comment (gigawhitlocks) [inline lib/apiservers/portlayer/swagger.json]: formatting?

File: lib/apiservers/portlayer/swagger.json [hunk 2]
```diff
@@ -1839,6 +1840,9 @@
 				"address": {
 					"type": "string"
 				},
+        "gateway": {
```
Comment (andrewtchin) [inline lib/apiservers/portlayer/swagger.json]: formatting?

File: lib/apiservers/portlayer/swagger.json (hunk 1 above)
Comment (jzt) [inline lib/apiservers/portlayer/swagger.json]: Fixed. It was emacs. +1 for vim.

File: lib/apiservers/portlayer/swagger.json (hunk 2 above)
Comment (jzt) [inline lib/apiservers/portlayer/swagger.json]: Fixed

**Comment by gigawhitlocks** [comment]
LGTM

**Comment by andrewtchin** [comment]
lgtm

**Comment by jzt** [comment]
♻️

**Pull request merged**